File: 1263/1263.cs
public class Solution {
    public int MinimumMovesToMoveABoxToTheirT(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(MinimumMovesToMoveABoxToTheirT(root.left), MinimumMovesToMoveABoxToTheirT(root.right));
    }
}

File: 1263/1263.go
func minimumMovesToMoveABoxToTheirT(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := minimumMovesToMoveABoxToTheirT(root.Left)
	r := minimumMovesToMoveABoxToTheirT(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}